build.gradle.kts

```diff
@@ -300,14 +301,11 @@ tasks.withType<KotlinCompile> {
 
 	kotlinOptions {
 
-		jvmTarget = "1.8"
+		jvmTarget = javaLangVersion.toString()
 
 		// enable experimental features so we can use the fancy ktor stuff
 		freeCompilerArgs += "-Xuse-experimental=kotlin.Experimental"
 		freeCompilerArgs += "-XXLanguage:+InlineClasses"
-
-		// use the newer (and less buggier?) compiler backend
-		useIR = true
```

Comment:
this option disappears in `1.6.0`

Reply:
Yeah, I think they removed the old compiler entirely in the newest major version, which apparently only a week old now. We've already been using the newer compiler for a while now, so we should be fine. Luckily the JVM backend is much more stable than the JS backend. I've had to avoid the new compiler for Kotlin/JS project because it wasn't quite working on my code yet. But JVM stuff seems fine.

Reply:
Nice, good to hear.